README.md

```diff
@@ -2,126 +2,161 @@
 
 > **This is not an official Google product.** This module is experimental and may not be ready for use.
 
-[![Build Status](https://travis-ci.org/GoogleCloudPlatform/stackdriver-errors-js.svg?branch=master)](https://travis-ci.org/GoogleCloudPlatform/stackdriver-errors-js)
-[![Dependency Status](https://david-dm.org/GoogleCloudPlatform/stackdriver-errors-js.svg)](https://david-dm.org/GoogleCloudPlatform/stackdriver-errors-js)
+[![Build Status][travis-ci status image]][travis-ci status link]
```

Comment:
I am not sure I see the benefits of moving the URLs at the bottom:

- as a reader in preview mode, it's the same
- as a reader in pain text, I now have to scroll back and forth to get to the link
- as a developer, it's error prone

Can you move the URLs back inline?

Reply:
@steren Thanks for the review!

Footnoting the links was mostly aimed at fixing two issues:
* In plain text view there was a lot of cruft, between warnings, status buttons, and inline links, before you actually get to the sentence saying what the project actually *is*.
* Editing (particularly rewording/moving things around) was a little annoying with long doc reference links mid paragraph.

Maybe it would be better to keep the button/thumbnail links down the bottom out of the way, but keep the things like youtube video link and key doc links inline for better usability in plain text form?

Any thoughts on the re-emphasis around use as a JS module? Was trying to cut down the repetition between the different methods of using the library without losing the specific details.

Reply:
I agree to move the button and thumbnail links at the bottom, but would like to keep the text hyperlinks in place.

Reply:
Done.